scipy/optimize/_differentialevolution.py

```diff
@@ -201,22 +199,92 @@ def differential_evolution(func, bounds, args=(), strategy='best1bin',
     .. [2] http://www1.icsi.berkeley.edu/~storn/code.html
     .. [3] http://en.wikipedia.org/wiki/Differential_evolution
     """
+    solver = DifferentialEvolution(func, bounds, args=args, strategy=strategy,
+                                   maxiter=maxiter, popsize=popsize, tol=tol,
+                                   mutation=mutation,
+                                   recombination=recombination, seed=seed,
+                                   polish=polish, init=init, atol=atol)
+    with solver as solver:
+        callback_termination, status_message = False, None
+        # Iterate on solver, which would go through either maxfev or maxiter.
+        for step in solver:
+            if disp:
+                print("differential_evolution step %d: f(x)= %g"
+                      % (step.nit, step.fun))
+            if (callback and
+                    callback(step.x,
+                             convergence=tol / solver.convergence) is True):
+                callback_termination = True
+                status_message = ('callback function requested stop early '
+                                  'by returning True')
+                break
+            if solver.converged():
+                break
+    result = solver.result
+    if callback_termination:
+        result.message = status_message
+        result.success = False
+    return result
+
+
+def _init_population_lhs(rng, num_population_members, population_shape,
+                         parameter_count):
+    """Initializes the population with Latin Hypercube Sampling.
+
+    Latin Hypercube Sampling ensures that each parameter is uniformly
+    sampled over its range.
+    """
+    # Each parameter range needs to be sampled uniformly. The scaled
+    # parameter range ([0, 1)) needs to be split into
+    # `num_population_members` segments, each of which has the following
+    # size:
+    segsize = 1.0 / num_population_members
+
+    # Within each segment we sample from a uniform random distribution.
+    # We need to do this sampling for each parameter.
+    samples = (segsize * rng.random_sample(population_shape)
+
+    # Offset each segment to cover the entire parameter range [0, 1)
```

Comment:
can you reformat this? blank line + comment in the middle of a single expression doesn't look right